src/raster/r.viewshed.exposure/testsuite/test_r_viewshed_exposure.py

```diff
@@ -0,0 +1,288 @@
+            "variance": 3.72542,
+            "coeff_var": 118.848259976884,
+            "sum": 196313,
+        },
+        "test_roads_F": {
+            "n": 120880,
+            "null_cells": 1904120,
+            "cells": 2025000,
+            "min": 0,
+            "max": 11.3565,
+            "range": 11.3565,
+            "mean": 1.20334,
+            "mean_of_abs": 1.20334,
+            "stddev": 1.47689,
+            "variance": 2.18121,
+            "coeff_var": 122.732631198116,
+            "sum": 145459.99071890,
+        },
+        "test_roads_D": {
+            "n": 120880,
+            "null_cells": 1904120,
+            "cells": 2025000,
+            "min": 0,
+            "max": 3.86,
+            "range": 3.86,
+            "mean": 0.118786,
+            "mean_of_abs": 0.118786,
+            "stddev": 0.275875,
+            "variance": 0.076107,
+            "coeff_var": 232.246,
+            "sum": 14358.8130874699,
+        },
+        "test_roads_S": {
+            "n": 120749,
+            "null_cells": 1904251,
+            "cells": 2025000,
+            "min": 0,
+            "max": 3.00867,
+            "range": 3.00867,
+            "mean": 0.0232121,
+            "mean_of_abs": 0.0232121,
+            "stddev": 0.202763,
+            "variance": 0.0411129,
+            "coeff_var": 873.523497082485,
+            "sum": 2802.8384487502,
+        },
+        "test_roads_V": {
+            "n": 120749,
+            "null_cells": 1904251,
+            "cells": 2025000,
+            "min": 0,
+            "max": 45.2416,
+            "range": 45.2416,
+            "mean": 0.34082,
+            "mean_of_abs": 0.34082,
+            "stddev": 3.80045,
+            "variance": 14.4434,
+            "coeff_var": 1115.09106861343,
+            "sum": 41153.6541857501,
+        },
+        "test_lakes": {
+            "n": 75396,
+            "null_cells": 1949604,
+            "cells": 2025000,
+            "min": 0,
+            "max": 4.61498,
+            "range": 4.61498,
+            "mean": 0.422226,
+            "mean_of_abs": 0.422226,
+            "stddev": 0.555438,
+            "variance": 0.308511,
+            "coeff_var": 131.549853569661,
+            "sum": 31834.1479829689,
+        },
+        "test_points": {
+            "n": 97340,
+            "null_cells": 1927660,
+            "cells": 2025000,
+            "min": 0,
+            "max": 11.8462,
+            "range": 11.8462,
+            "mean": 0.0432476,
+            "mean_of_abs": 0.0432476,
+            "stddev": 0.144737,
+            "variance": 0.0209488,
+            "coeff_var": 334.670567638394,
+            "sum": 4209.72256931104,
+        },
+    }
+
+    @classmethod
+    def setUpClass(cls):
+        """Save the current region
+        We cannot use temp_region as it is used by the module.
+        """
+
+        # Save current region to temporary file
+        cls.runModule("g.region", flags="u", save="{}_region".format(cls.tempname))
```

Comment:
I don't see why use_temp_region wouldn't work here:

```suggestion
        gs.use_temp_region()
        gs.run_command("g.region", raster=cls.dsm)
```

Reply:
We tried that earlier but it failed with raster2numpy or numpy2raster (don`t remember)...
But lets try again with G8 in 43116a4 ...